api/v1/models/product.py

```diff
@@ -23,3 +26,18 @@ class Product(BaseModel, Base):
     name = Column(String, nullable=False)
     description = Column(Text, nullable=True)
     price = Column(Numeric, nullable=False)
+    created_at = Column(DateTime(timezone=True), server_default=func.now())
+    updated_at = Column(DateTime(timezone=True), server_default=func.now(), onupdate=func.now())
+
+
+
+class Category(Base):
+    __tablename__ = "categories"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(100), unique=True, index=True)
+    description = Column(Text)
+    slug = Column(String(100), unique=True, index=True)
+    parent_id = Column(Integer, ForeignKey("categories.id"), nullable=True)
+
+    children = relationship("Category")
```

Comment:
It's obvious you didn't ask for permission to create this table. The `created_at` and `updated_at` already exists through the `BaseModel` class. Ask and you would be guided or it would be done for you.